src/cephadm/cephadm

```diff
@@ -3395,6 +3400,10 @@ def _get_parser():
         '--skip-dashboard',
         action='store_true',
         help='do not enable the Ceph Dashboard')
+    parser_bootstrap.add_argument(
+        '--dashboard-password-noupdate',
+        action='store_true',
+        help='stop forced dashboard password change')
```

Comment:
Given the existing flag for dashboard refers to a positive action ("Require Update"), and given the default behaviour hasn't changed, why not keeping that convention (I often find confusing "negative" flags)?
```suggestion
        '--dashboard-password-update',
        action='store_true',
        help='force dashboard password change on first logon')
```

Reply:
my reasoning for the negative flag was that we want the user the change there password after bootstrap so having it do that by default seemed better and if they really don't want to change it they can use the flag.